contrib/macdeploy/background.svg

```diff
@@ -1,34 +1,26 @@
-<?xml version="1.0" standalone="no"?>
-<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
- "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
-<svg version="1.0" xmlns="http://www.w3.org/2000/svg" width="1000pt" height="640pt" viewBox="0 0 1000 640" preserveAspectRatio="xMidYMid meet">
-	<!-- kate: space-indent off;
-	Copyright (c) 2015 The Bitcoin Core developers
-	Distributed under the MIT software license, see the accompanying
-	file COPYING or http://www.opensource.org/licenses/mit-license.php.
```

Comment:
What I meant was a copyright for the SVG itself, like you see here

Reply:
ok - I will include the copyright header in the background.svg file.

Reply:
@luke-jr - is this an  "<!-- kate: space-indent off;   ... -->" artifact from an old linting tool - or do these need to stay?

Reply:
The new svg files use spaces. So the kate header was removed.